weather/www/js/app.js

```diff
@@ -0,0 +1,147 @@
+var app = angular.module('weather', ['ionic']);
+
+
+app.controller('HomeCtrl', function($scope, weatherService) {
+
+  
+
+  weatherService.get().then(function(response) {
+
+
+//got this idea from stack overflow but couldn't find the link
+    var monthNames = [
+      "January", "February", "March",
+      "April", "May", "June", "July",
+      "August", "September", "October",
+      "November", "December"
+    ];
+
+    var days = [
+      'Sunday', 'Monday', 'Tuesday', 'Wednesday',
+      'Thursday', 'Friday', 'Saturday'
+    ];
+
```

Comment:
This is exactly what I did for days of the week. Kind of a pain, I would think it would be a default part of JS.